clusterloader2/testing/density/config.yaml

```diff
@@ -6,6 +6,8 @@
 {{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 100}}
 {{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 30}}
 {{$TEST_THROUGHPUT := DefaultParam .TEST_THROUGHPUT 20}}
+{{$LATENCY_POD_CPU := DefaultParam .LATENCY_POD_CPU 80}}
+{{$LATENCY_POD_MEMORY := DefaultParam .LATENCY_POD_MEMORY 300}}
```

Comment:
I increased memory request + I added comments.

The values will be different then ones used in original tests. We try to fit 5 pods instead of just one per node.

Reply:
Small changes again. Currently latency pods allocate 10% of 1cpu and 4GB node.